app/cluster/state/static.go

```diff
@@ -37,10 +38,11 @@ func (s *StaticProvider) ServerMode(ctx context.Context) <-chan servermode.Chang
 }
 
 // WorkspaceIDs returns an empty channel, since we don't expect workspaceIDs updates with static provider
-// TODO: This method should return proper workspaceIDs for backend config, even with static provide.
 func (s *StaticProvider) WorkspaceIDs(ctx context.Context) <-chan workspace.ChangeEvent {
-	ch := make(chan workspace.ChangeEvent)
-
+	ch := make(chan workspace.ChangeEvent, 1)
+	wId := backendconfig.DefaultBackendConfig.AccessToken()
```

Comment:
We should avoid accessing objects globally. It makes tests really hard. Also, what if we want to use a non-default backend config? I think the backendConfig should be a `StaticProvider` attribute inside the `struct`.